### DISCUSSION TITLE
error message

### DISCUSSION THREAD

**Discussion by BenVoyons** [utmapp/UTM#6379] [macOS]
hi, i cant open my W11 with my UTM virtual machine. I have this error....

QEMU error: QEMU a quitté avec l’erreur : qemu-aarch64-softmmu: -drive if=none,media=cdrom,id=driveEC8187BB-6F73-4788-917C-50013BC6F4F3,file=/Users/iMacMichel/Library/Mobile Documents/.Trash/22631.2861.231204-0538.23H2_NI_RELEASE_SVC_REFRESH_CLIENTCONSUMER_RET_A64FRE_fr-ca.iso,readonly=on: Could not open '/Users/iMacMichel/Library/Mobile Documents/.Trash/22631.2861.231204-0538.23H2_NI_RELEASE_SVC_REFRESH_CLIENTCONSUMER_RET_A64FRE_fr-ca.iso': Operation not permitted

Solution please, thanks

iMac 2021 M1
Sonoma 14.5

**Comment by fredpixlab** [comment]
ok got it .. 

Your windows file is on iCloud that put by default files .. offline .. for your iso to be downloaded and it's working again
